playbooks/roles/ocp-descheduler/tasks/main.yaml

```diff
@@ -0,0 +1,177 @@
+---
+## descheduler deployment and e2e run tasks
+
+# Environment variables check block
+- set_fact:
+    descheduler_github_username: "{{ lookup('ansible.builtin.env', 'GITHUB_USERNAME') }}"
+    descheduler_github_access_token: "{{ lookup('ansible.builtin.env', 'GITHUB_ACCESS_TOKEN') }}"
+    descheduler_e2e_dir: "{{ [descheduler_workdir, descheduler_e2e_repo.split('/')[-1]] | join('/') }}"
+
+- fail:
+    msg: "Please set descheduler_e2e_repo variable with the e2e repo URL."
+  when: descheduler_e2e_repo == "" or descheduler_e2e_repo == None
+
+- fail:
+    msg: "Please set the environment variables GITHUB_USERNAME and GITHUB_ACCESS_TOKEN on your host."
+  when: descheduler_github_username == "" or descheduler_github_access_token == ""
+
+# Cluster health check block
+- name: Check the cluster health
```

Comment:
there is an existing role [check-cluster-health](https://github.com/ocp-power-automation/ocp4-playbooks-extras/tree/main/playbooks/roles/check-cluster-health). Can you re-use that?

Reply:
Used the existing role to check the cluster health status